src/utils/utils.ts

```diff
@@ -0,0 +1,10 @@
+const timeConvert = (n: number) => {
```

Comment:
Let's take a look on date-fns library. It's really useful tool for operating with dates